data-model/fdo-type/shared-model/0.3.0/examples/tombstone-metadata-example.json

```diff
@@ -1,7 +1,7 @@
 {
   "@type": "ods:Tombstone",
   "ods:tombstonedDate": "2023-06-15T14:30:00Z",
```

Comment:
Shouldn't this also be `ods:tombstoneDate` and the `ods:tombstoneByAgent` ?

Reply:
I did a replace all :upside_down_face: reverted